Datahub.Web/Handlers/SitemapController.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Text;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Datahub.Web;
+using Datahub.Web.Data;
+using Microsoft.Extensions.Caching.Memory;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http.Extensions;
+
+public static class CacheKeys
+{
+    public static string RobotsTxt { get { return "_RobotsTxt"; } }
+    public static string Sitemap { get { return "_Sitemap"; } }
```

Comment:
The idea here was to make it clear that these are only every related to the cache and nothing else, pushing them into the other class needs comments as to what they are but here its fairly obvious why and where they are to be used. Happy to do otherwise if you really want it.

Reply:
Other point here is that the cache is a global service

Reply:
Moved out of cachekeys object into controller as on retrospect these aren't used outside the controller as was my original intention